package.json

```diff
@@ -8,7 +8,8 @@
     "build": "make build",
     "fix": "make fix",
     "lint": "make lint",
-    "test": "make test"
+    "test": "make test",
+    "version": "yarn --immutable-cache && git add yarn.lock"
```

Comment:
`--immutable-cache` is used here to bail when new dep needs to be _fetched_ from network, which can happen when people edit external deps in `package.json` but forget to checkout the updated yarn.lock, and if that happens it should be addressed in separate commits instead of the version commit. In all, the version commit should include the following changes only:
- Updated version in related package.json
- Updated `workspace@new-version` resolution in yarn.lock